OpenRA.Mods.Common/Widgets/Logic/ServerCreationLogic.cs

```diff
@@ -147,7 +148,7 @@ void CreateAndJoin()
 				return;
 			}
 
-			ConnectionLogic.Connect(IPAddress.Loopback.ToString(), Game.Settings.Server.ListenPort, password, onCreate, onExit);
+			ConnectionLogic.Connect(new ConnectionAddress("localhost", Game.Settings.Server.ListenPort), password, onCreate, onExit);
```

Comment:
why is the change needed? ```"localhost```
```IPAddress.Looback``` is ipv6 aware?!

Reply:
IPAddress.Loopback is IPv4 only according to the documentation. That would obviously fail on an IPv6 only host.

Reply:
then use IPv6Loopback

Reply:
And where do you get `address` from?